File: sync_js.js
const fs = require('fs');
const js = fs.readFileSync('src/JavaScript.js', 'utf8');
const html = `<script>\n${js}\n</script>`;
fs.writeFileSync('JavaScript.html', html, 'utf8');
console.log('Synced src/JavaScript.js to JavaScript.html successfully!');

File: sync_js.py
import io

with io.open('src/JavaScript.js', 'r', encoding='utf-8') as f:
    content = f.read()

html_content = '<script>\n' + content + '\n</script>'

with io.open('JavaScript.html', 'w', encoding='utf-8') as f:
    f.write(html_content)

print("[OK] Synced src/JavaScript.js -> JavaScript.html")
